packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -130,22 +131,36 @@ export class AtomicBreadcrumbManager {
     return breadcrumbs;
   }
 
-  private getRangeBreadrumbValues(values: Breadcrumb<RangeFacetValue>) {
+  private getRangeBreadrumbValues(
+    values: Breadcrumb<RangeFacetValue>,
+    needDateFormatting: boolean
+  ) {
     const {breadcrumbsToShow, moreButton} = this.collapsedBreadcrumbsHandler(
       values
     );
-    const renderedBreadcrumbs = breadcrumbsToShow.map((breadcrumbValue) => (
-      <li part="breadcrumb-value" class="pr-3">
-        <button
-          part="breadcrumb-button"
-          class={this.buttonClasses}
-          onClick={breadcrumbValue.deselect}
-        >
-          {breadcrumbValue.value.start} - {breadcrumbValue.value.end}
-          {this.mainClear}
-        </button>
-      </li>
-    ));
+    const renderedBreadcrumbs = breadcrumbsToShow.map((breadcrumbValue) => {
+      let ariaLabel: string;
+      if (needDateFormatting) {
+        const dateStart = new Date(breadcrumbValue.value.start);
+        const dateEnd = new Date(breadcrumbValue.value.end);
+        ariaLabel = `Remove inclusion filter on ${dateStart.toLocaleString()} to ${dateEnd.toLocaleString()}`;
```

Comment:
This reads the date correctly. But it changes the order of the date( e.g. `2008/01/01@00:00:00` becomes `1/1/2008, 12:00:00 AM`). Is it important to keep the dates in their original order? (e.g `2008/1/1, 12:00:00 AM`)

Reply:
I would say correct reading of the date is more important than the order of the numbers.